src/lib/utilities/route-for-api.ts

```diff
@@ -1,16 +1,11 @@
-// import { browser } from '$app/env';
-// import { namespaceUrlPattern } from './namespace-url-pattern';
-
-// const RealBaseUrl = `https://web.${
-//   namespaceUrlPattern.match(window.location.pathname)?.namespace
-// }.tmprl.cloud/api/v1/`;
-
-let base = (import.meta.env?.VITE_API as string) ?? process.env.VITE_API;
+let base =
+  import.meta.env?.VITE_API ??
```

Comment:
This will never resolve because of ?. When this packages does process.env.VITE_API resolve to ""?

I think globalThis needs to come first to have our correct functionality.